pppd/options.c

```diff
@@ -332,7 +333,10 @@ struct option general_options[] = {
       OPT_A2PRINTER | OPT_NOPRINT, (void *)user_unsetprint },
 
     { "defaultroute-metric", o_int, &dfl_route_metric,
```

Comment:
Need to add o_uint option.

Reply:
Why? Do we need to allow values >= 2^31?

Reply:
To match kernel unsigned option here.  To be realistic I highly doubt a value >=2^31 makes sense, so let's just leave this as is for now.